docs/getting-started-guides/kops.md

```diff
@@ -0,0 +1,158 @@
+
+* Create the S3 bucket using `aws s3 mb s3://clusters.dev.example.com`
+
+* You can `export KOPS_STATE_STORE=s3://clusters.dev.example.com` and then kops will use this location by default. 
+   We suggest putting this in your bash profile or similar. 
+
+
+### (4/5) Build your cluster configuration
+
+Run "kops create cluster" to create your cluster configuration:
+
+`kops create cluster --zones=us-east-1c useast1.dev.example.com`
+
+kops will create the configuration for your cluster.  Note that it _only_ creates the configuration, it does
+not actually create the cloud resources - you'll do that in the next step with a `kops update cluster`.  This
+give you an opportunity to review the configuration or change it.
+
+It prints commands you can use to explore further:
+
+* List your clusters with: `kops get cluster`
+* Edit this cluster with: `kops edit cluster useast1.dev.example.com`
+* Edit your node instance group: `kops edit ig --name=useast1.dev.example.com nodes`
+* Edit your master instance group: `kops edit ig --name=useast1.dev.example.com master-us-east-1c`
+
+If this is your first time using kops, do spend a few minutes to try those out!  An instance group is a
+set of instances, which will be registered as kubernetes nodes.  On AWS this is implemented via auto-scaling-groups.
+You can have several instance groups, for example if you wanted nodes that are a mix of spot and on-demand instances, or
+GPU and non-GPU instances.
+
+
+### (5/5) Create the cluster in AWS
+
+Run "kops update cluster" to create your cluster in AWS:
+
+`kops update cluster useast1.dev.awsdata.com --yes`
+
+That takes a few seconds to run, but then your cluster will likely take a few minutes to actually be ready.
+`kops update cluster` will be the tool you'll use whenever you change the configuration of your cluster; it
+applies the changes you have made to the configuration to your cluster - reconfiguring AWS or kubernetes as needed.
+
+For example, after you `kops edit ig nodes`, then `kops update cluster --yes` to apply your configuration, and
+sometimes you will also have to `kops rolling-update cluster` to roll out the configuration immediately.
+
+Without `--yes`, `kops update cluster` will show you a preview of what it is going to do.  This is handy
+for production clusters!
+
+### Explore other add-ons
+
+See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+
+## What's next
+
+* Learn more about [Kubernetes concepts and kubectl in Kubernetes 101](/docs/user-guide/walkthrough/).
+* Learn about `kops` [advanced usage](https://github.com/kubernetes/kops)
+
+## Cleanup
+
+* To delete you cluster: `kops delete cluster useast1.dev.example.com --yes`
+
+## Feedback
+
+* Slack Channel: [#sig-aws](https://kubernetes.slack.com/messages/sig-aws/) has a lot of kops users
```

Comment:
I think that makes sense.  When/if we find that the kops discussion is primarily about kops & lifecycle, and not about kops & k8s & AWS then we can revisit.